site/pages/our-work/case-study-fortnum-and-mason-digital-transformation-slice/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import styles from './style.css';
+import Link from '../../../components/link';
+
+import fmLogo from './images/fortnum-mason-logo.png';
+
+export default function CaseStudyFortnumDigitalTransformationSlice() {
+  return (
+    <div className={styles.caseStudyContainer}>
+      <div className={styles.caseStudyContent}>
+        <div className={styles.caseStudyTextContainer}>
+          <Link to="fMTeaCaseStudy">
+            <img src={fmLogo} className={styles.clientLogo} alt="Fortnum and Mason logo" />
+            <h2 className={styles.caseStudyTextContainerHeader}>
+              The ongoing digital transformation of a 310-year old retailer
+            </h2>
+            <p className={styles.description}>
+              Discover how we helped customers find products faster and drive sales of tea.
+            </p>
+            <div className={styles.links}>
+              <p className={styles.readmore}>Read more</p>
+            </div>
```

Comment:
if this is an interactive item, could we make it a `button` or an `a` tag? Semantic HTML is important for screenreaders and users who navigate using the keyboard

Reply:
The whole thing is wrapped inside a `<Link />` tag, which is rendered as an `<a />` tag, so I think this sould be fine.

Reply:
Or would it be better for accessibility to wrap each paragraph in a separate link instead of the whole thing? I don't know, to be honest.

Reply:
Screenreaders take anything they identify as a link out of context and into one link. So if it just read out this block, nothing else, would it make sense?
I don't think we should do separate links, because 'Read more' by itself doesn't make any sense.

Reply:
I just had a listen, I think it'll be ok as is!